### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tbazant**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Vale-based technical documentation style guide
 
 A [Vale](https://vale.sh)-compatible style for technical writers documenting open-source software and for anyone writing about Open Source.
```
Comment (dariavladykina) [inline README.md:3]:
```suggestion
A [Vale](https://vale.sh)-compatible style for technical writers documenting open source software and for anyone writing about Open Source.
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
+The style closely follows [SUSE Documentation Style Guide][https://documentation.suse.com/style/current/html/docu_styleguide/index.html]
+an is developed by the SUSE Documentation Team.
```
Comment (dariavladykina) [inline README.md:5]:
```suggestion
The style guide closely follows the [SUSE Documentation Style Guide][https://documentation.suse.com/style/current/html/docu_styleguide/index.html]
and is developed by the SUSE Documentation Team.
```

File: README.md [hunk 3]
```diff
@@ -86,13 +88,13 @@ StylesPath = styles
 MinAlertLevel = suggestion
 [asciidoctor]
 experimental = YES
-[*.{md,txt,adoc}]
-BasedOnStyles = suse-vale-styleguide
+[*]
+BasedOnStyles = common
 ```
 
 Save the changes.
```
Comment (dariavladykina) [inline README.md:95]:
```suggestion
3. Save the changes.
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see my suggestions here. Thanks!

File: README.md [hunk 4]
```diff
@@ -86,13 +88,13 @@ StylesPath = styles
...
-BasedOnStyles = suse-vale-styleguide
+[*]
+BasedOnStyles = common
 ```
 
-Save the changes.
+3. Save the changes.
 
-## Add XML support (openSUSE)
+## Add XML and AsciiDoc support (openSUSE)
```
Comment (tomschr) [inline README.md:97]: Maybe be a bit more specific? I guess you only support _DocBook_-XML and not other XML dialects, right? Therefor, I'd suggest:

```suggestion
## Add DocBook-XML and AsciiDoc support (openSUSE)
```

**Review by tomschr — COMMENTED** [review]
Just a small suggestion, thanks Tomas! :+1:

**Pull request merged**